RELEASING.md

```diff
@@ -77,28 +79,28 @@ After all [pull requests](https://github.com/GoogleCloudPlatform/google-cloud-ru
     $ git push <remote> <gem>/v<version>
     ```
 
-1. Repeat steps 1 through 15 if you're releasing multiple gems.
+1. Wait until the [Circle CI build](https://circleci.com/gh/GoogleCloudPlatform/google-cloud-ruby) has passed for the tag.
 
-1. Wait until the tag build job from step 15 has started (preferably wait for it to finish and turn green). Then push the commit to the master branch. This will trigger another [Circle CI](https://circleci.com/gh/GoogleCloudPlatform/google-cloud-ruby) build on master branch.
+1. Confirm that the new version is displayed on the [google-cloud-ruby gh-pages doc site](http://googlecloudplatform.github.io/google-cloud-ruby/), both in the packages pulldown and the version switcher.
 
-    ```sh
-    $ git push <remote> master
-    ```
+   If the gh-pages doc site has not been updated, inspect the build logs to confirm that the release task completed successfully, and that the docs build succeeded. This can still fail even on a green build because it is an "after" action in the build.
+
+1. Confirm that the gem for the new version is available on [RubyGems.org](https://rubygems.org/gems/google-cloud).
 
 1. On the [google-cloud-ruby releases page](https://github.com/GoogleCloudPlatform/google-cloud-ruby/releases), click [Draft a new release](https://github.com/GoogleCloudPlatform/google-cloud-ruby/releases/new). Complete the form. Include the bullet-point lists of the major and minor changes from the gem's `CHANGELOG.md`. You can also add examples, fixes, thank yous, and anything else helpful or relevant. See google-cloud-node [v0.18.0](https://github.com/GoogleCloudPlatform/google-cloud-node/releases/tag/v0.18.0) for an example with all the bells and whistles.
 
 1. Click `Publish release`.
 
-1. Check again that the [Circle CI build](https://circleci.com/gh/GoogleCloudPlatform/google-cloud-ruby) has passed for the tag. Also, inspect the build logs to confirm that the `release` task completed successfully, and that the docs build succeeded. This can fail on a green build because it is an "after" action in the build.
-
-1. Confirm that the gem for the new version is available on [RubyGems.org](https://rubygems.org/gems/google-cloud).
+1. Repeat steps 1 through 20 if you are releasing multiple gems.
```

Comment:
Thanks -- I see now this is what you were trying to tell me on chat yesterday...